src/app/components/feed/SaveFeed.js

```diff
@@ -158,16 +156,11 @@ const SaveFeed = (props) => {
   const [title, setTitle] = React.useState(feed.name || '');
   const [description, setDescription] = React.useState(feed.description || '');
   const [selectedListId, setSelectedListId] = React.useState(isFeedOwner ? feed.saved_search_id : feedTeam.saved_search_id);
-  const [discoverable, setDiscoverable] = React.useState(Boolean(feed.discoverable));
+  const [discoverable] = React.useState(Boolean(false));
```

Comment:
@brianfleming , minor change, since we're not changing `discoverable`, it doesn't need to be a state - you can change this line simply to `const discoverable = false`.